resources/aws/kms.go

```diff
@@ -21,14 +27,31 @@ func (kk *KMSKey) CreateOrFail() error {
 	if err != nil {
 		return microerror.MaskAny(err)
 	}
+
+	if _, err := kk.Clients.KMS.CreateAlias(&kms.CreateAliasInput{
+		// Alias names need to start from "alias/" prefix.
+		AliasName:   aws.String(kk.fullAlias()),
+		TargetKeyId: key.KeyMetadata.Arn,
+	}); err != nil {
+		return microerror.MaskAny(err)
+	}
+
 	kk.arn = *key.KeyMetadata.Arn
+
 	return nil
 }
 
-func (kms *KMSKey) Delete() error {
-	return microerror.MaskAny(notImplementedMethodError)
+func (kk *KMSKey) Delete() error {
+	// AWS API doesn't allow to delete the KMS key itself, but we remove the alias and leave the key nameless.
```

Comment:
Can't we use [`ScheduleKeyDeletion`](https://docs.aws.amazon.com/sdk-for-go/api/service/kms/#KMS.ScheduleKeyDeletion) at least, so the key will eventually be deleted?